src/button.tsx

```diff
@@ -20,3 +22,85 @@ export const ProtoButton = React.forwardRef<HTMLButtonElement, JSX.IntrinsicElem
     {...rest}
   >{children}</button>,
 );
+
+/** The kind of buttons a "Button" can be. Used for styling */
+export type Kind = "normal" | "danger" | "call-to-action";
+
+type ButtonProps = JSX.IntrinsicElements["button"] & {
+    kind?: Kind;
+    extraCss?: Interpolation<Theme>;
+};
+
+/** A styled button */
+export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ kind = "normal", extraCss, children, ...rest }, ref) => (
+    <button ref={ref} type="button" css={css(kind, extraCss)} {...rest}>{children}</button>
+  ),
+);
+
+/**
+ * Returns css for different types of buttons.
+ * Comes in the kinds "normal", "danger" and "happy".
```

Comment:
"happy" is outdated here, but maybe you can also just remove the second sentence, as that information can easily be obtained looking at `Kind`